### ISSUE TITLE
static route at the root path does not work as expected

### ISSUE THREAD

**Issue by LoganTann** [l2ig/aqua#93]
Hello, in a first place, thanks to the dev, this is a very nice library !

I'm trying to embed a basic front-end with aqua. I created the following code : 

```js
const app: Aqua = new Aqua(3000);
app.serve("web-front", "/");
```

Expected Behavior : Going to `localhost:3000/index.html` should show file web-front/index.html

Current Behavior:
* `localhost:3000/index.html` produces a 404 error.
* `localhost:3000//index.html` does the expected result.

**Comment by felixfong227** [comment]
from my understanding, the cost of this issue is because of a missing "/" when reading the file from the disk, and I think I got it fixed in #94 (🤞

**Comment by l2ig** [comment]
Fix included in v1.3.4
